code/modules/trade/datums/trade_stations_presets/4-unique/illegalbeacon2.dm

```diff
@@ -51,5 +51,6 @@
 		)
 	)
 	offer_types = list(
+		/obj/machinery/nuclearbomb = offer_data("nuclear weapon", 50000, 1),
```

Comment:
Really tried to sneak this shit in huh

Reply:
BASED
KEEP IT IN

Reply:
Tbf I thought it fell under "adds a lot of new offers" but I probably should have flagged it directly in the changelog

Reply:
Dude, no. Just no.

Reply:
I'm not kidding, remove this.